cfgov/unprocessed/css/pages/beta.less

```diff
@@ -13,10 +13,14 @@
     font-size: 0.875em;
     transition: padding 0.2s ease-out;
 
-    &.expandable__expanded {
+    &.m-expandable__expanded {
         padding: @grid_gutter-width 0;
     }
 
+    .m-expandable_content {
+        border-bottom: none;
```

Comment:
Gotcha, well that's fine. Like mentioned earlier we'll likely want to abstract this out of the expandables at some point. Alternatively if the padding could be placed on the content inside the expandable then the border could end up at the edge.